### PR TITLE
Adding assembly_stats to conda-forge

### PR DESCRIPTION
**Pull request by MikeTrizna**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by MikeTrizna** [comment]
I'm commenting here to satisfy the final task list: "GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there"

File: recipes/assembly_stats/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "assembly_stats" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "4393e3007a6d8674ff32348ae33a5e013a38db5452e86ab4b4b86883aab63e96"
```
Comment (marcelotrevisani) [inline recipes/assembly_stats/meta.yaml]: Could you please remove the quotes from the ``sha256``?
```suggestion
  sha256: 4393e3007a6d8674ff32348ae33a5e013a38db5452e86ab4b4b86883aab63e96
```

File: recipes/assembly_stats/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+build:
+  number: 0
+  entry_points:
+    - assembly_stats=assembly_stats.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/assembly_stats/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/assembly_stats/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "4393e3007a6d8674ff32348ae33a5e013a38db5452e86ab4b4b86883aab63e96"
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/assembly_stats/meta.yaml]: I think this recipe can be ``noarch``
```suggestion
  number: 0
  noarch: python
```

**Pull request merged**